lib/sda/service_test.go

```diff
@@ -444,6 +449,34 @@ func TestServiceBackForthProtocol(t *testing.T) {
 	assert.Equal(t, resp.Val, 10)
 }
 
+func TestClient_Send(t *testing.T) {
```

Comment:
Yes, running `egrep -r "func.*Test.*_.*\(" .` on the go-source doesn't give back as many examples for Tests including '_' as I thought, so this argument doesn't hold.

As it's an issue of style, I propose we just vote on it (nice we are three ;) - no blank bulletins. I like the idea of having `TestT_M`, so +1 for me.